<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="5,5,100">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="8,10">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_serialport">
          <property name="maximumSize">
           <size>
            <width>78</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="text">
           <string>Serial Port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="port_list"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLineEdit" name="lineEdit_savepath">
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="text">
           <string>Entry Lock Pressure Log.csv</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="AutoSaveLog">
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="text">
           <string>Auto Save Log</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="70,30">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QComboBox" name="gaugeID">
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <item>
           <property name="text">
            <string>Entry Lock</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>INFINITY</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_pressure">
          <property name="text">
           <string>Pressure</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_time">
          <property name="text">
           <string>Time</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_dynamic_or_log">
          <item>
           <property name="text">
            <string>Dynamic Data</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Log Data</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_start">
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_clear">
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_mpl"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
